a2a/1.0/modules/ROOT/pages/a2a-connector-studio.adoc

```diff
@@ -0,0 +1,138 @@
+= Using Anypoint Studio to Configure {connector-name} Connector {version}
+:exchange-search-name: a2a
+:connector-name: A2A
+:version: 0.1.0-BETA
+:lc-connector-name: a2a
+:studio-connector-name: A2A Connector
+:tls:
+
+include::connectors::partial$t-studio-partial.adoc[tags=topic-intro]
+
+
+[[create-mule-project]]
+== Create a Mule Project
+
+include::connectors::partial$t-studio-partial.adoc[tags=create-project-intro]
+
+[[add-connector-to-project]]
+== Add the Connector to Your Mule Project
+
+include::connectors::partial$t-studio-partial.adoc[tags=add-connector-to-project]
+
+[[configure-source]]
+== Configure a Source
+
+include::connectors::partial$t-studio-partial.adoc[tags=configure-source-intro]
+You can configure one of these sources to use with {connector-name} Connector:
+
+* *Task Listener*
++
+This source is an extension of HTTP Listener. It listens for incoming tasks directed towards its agent, such as messages or API calls.
++
+include::connectors::partial$t-studio-partial.adoc[tags=listener-scheduler-descriptions]
+
+
+include::connectors::partial$t-studio-partial.adoc[tags=listener-steps]
+
+[[add-connector-operation]]
+== Add a Connector Operation to the Flow
+
+include::connectors::partial$t-studio-partial.adoc[tags=add-connector-operation]
+
+[[configure-global-element]]
+== Configure a Global Element for the Connector
+
+include::connectors::partial$t-studio-partial.adoc[tags=global-element-intro]
+
+include::connectors::partial$t-studio-partial.adoc[tags=property-placeholders]
+
+include::connectors::partial$t-studio-partial.adoc[tags=global-element-first-steps]
+
+. In the *General* tab, in *Connection*, select the authentication method to configure:
+* <<expression-or-bean-ref>>
+* <<basic_authentication>>
+* <<digest>>
+* <<ntlm>>
+
+ifdef::tls[]
+include::connectors::partial$t-studio-partial.adoc[tags=global-element-tls-steps]
+endif::[]
+
+[[expression-or-bean-ref]]
+=== Expression or Bean Reference
+
+Uses a DataWeave expression or bean reference to define authentication.
+
+Enter the authentication information in the *Global Element Properties* screen:
+
+. Select *General > Authentication* and enter or set the values for these fields:
```

Comment:
which fields? I feel like you can remove the "Select General > Authentication and enter or set the values for these fields:" step from all four auth types (or combine it with next step)